networkx/linalg/laplacianmatrix.py

```diff
@@ -74,7 +106,6 @@ def laplacian_matrix(G, nodelist=None, weight="weight"):
     return D - A
 
 
-@not_implemented_for("directed")
```

Comment:
I also added the capability to use `normalized_laplacian_matrix` on DiGraphs.

Reply:
Nice!